charts/telepresence/README.md

```diff
@@ -27,30 +27,45 @@ The following tables lists the configurable parameters of the Ambassador chart a
 
 | Parameter                | Description                                                                                                             | Default                                                                                           |
 |--------------------------|-------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|
-| image.repository         | The repository to download the image from. Set `TELEPRESENCE_REGISTRY=image.repository` locally if changing this value. | `docker.io/datawire/tel2`                                                                         |
+| image.registry         | The repository to download the image from. Set `TELEPRESENCE_REGISTRY=image.registry` locally if changing this value. | `docker.io/datawire`                                                                         |
+| image.name         | The name of the image to use for the traffic-manager                                                                      | `tel2`                                                                                            |
 | image.pullPolicy         | How the `Pod` will attempt to pull the image.                                                                           | `IfNotPresent`                                                                                    |
 | image.tag                | Override the version of the Traffic Manager to be installed.                                                            | `""` (Defined in `appVersion` Chart.yaml)                                                         |
 | image.imagePullSecrets   | The `Secret` storing any credentials needed to access the image in a private registry.                                  | `[]`                                                                                              |
+| ports            | The ports the Traffic Manager `Deployment` will expose. **Do not change.**                               | `[{"name":"api","port":8081},{"name": "https", "port": 8443]` |
```

Comment:
What's the point in having a setting if we tell people not to set it?

Reply:
I saw that this is what we did with service.ports, but I think it doesn't really make sense / is kinda confusing, so removing the "do not changes" I added, as well as the pre-existing ones and will have it hard-coded in the yaml.